src/utils/upload.rs

```diff
@@ -37,10 +42,29 @@ pub async fn upload_dependencies<'a>(
         let output = punfile.configuration.output.clone();
         let cache_prefix = punfile.configuration.prefix.clone();
         let bucket_name = punfile.configuration.s3_bucket.clone();
+        let default_repo = Repository {
+            name: "".to_string(),
+            repo_name: "".to_string(),
+            version: "".to_string()
+        };
+
+        let expanded_frameworks: Vec<&str> = frame.split(".").collect();
+
+
+        let framework_key = expanded_frameworks.get(0).unwrap();
+
+        let framework = punfile.frameworks.iter().find(|item| item.name.contains(framework_key)).unwrap_or(&default_repo);
+        println!("discovered framework version {}", framework.name);
```

Comment:
since `version` here can be an empty string can you do a conditional and just append to the next `println`
something like
```swift
if version.isEmpty {
  println("Found {}/{}" , ...)
} else {
  println("Found {}/{} @ Version {}", ...)
}
```